validator/schema.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
We shouldn't have two copies of the schema. Probably best to have one top-level schema def and then use a symlink from within the Python package.

Reply:
Yes, I tested with a symlink but did a revert (https://github.com/opengeospatial/geoparquet/pull/58/commits/04f1bddacc704a5738734ecf354a347aba3f9f45), because setup.py does not find it. It works with `pip install -e .` but not with `pip install .`. I'll keep it only in the python package to merge, and we can figure out what's the best way to share it later.